src/hooks/useCreateEmptyReportConfirmation.tsx

```diff
@@ -1,8 +1,11 @@
 import type {ReactNode} from 'react';
 import React, {useCallback, useMemo, useState} from 'react';
+import {View} from 'react-native';
+import CheckboxWithLabel from '@components/CheckboxWithLabel';
 import ConfirmModal from '@components/ConfirmModal';
 import Text from '@components/Text';
 import TextLink from '@components/TextLink';
+import useThemeStyles from '@hooks/useThemeStyles';
```

Comment:
Please fix the eslint

Reply:
Can't see any ESLint issue here, can you check again?